pkg/sync/v5/data/Service/default/router-stats.yaml

```diff
@@ -1,16 +1,13 @@
 apiVersion: v1
 kind: Service
 metadata:
-  annotations:
-    prometheus.io/port: "1936"
-    prometheus.io/scrape: "true"
```

Comment:
why were these removed?

Reply:
these were remnants from a previous implementation of prometheus scraping, not used in this one.